### PR TITLE
Persist LTI Keys

### PR DESCRIPTION
**Pull request by gabrielfs7**
How to persist LTI keys:

To start testing add content to path:

```
data/ltiKeyChain/platform/default/private.key
```
and

```
data/ltiKeyChain/platform/default/public.key
```

https://oat-sa.atlassian.net/browse/NEX-1533

### CODE REVIEW

File: models/classes/Security/DataAccess/Repository/PlatformKeyChainRepository.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+use oat\tao\model\security\Business\Contract\KeyChainRepositoryInterface;
+use oat\tao\model\security\Business\Domain\Key\Key;
+use oat\tao\model\security\Business\Domain\Key\KeyChain;
+use oat\tao\model\security\Business\Domain\Key\KeyChainCollection;
+use oat\tao\model\security\Business\Domain\Key\KeyChainQuery;
+
+class PlatformKeyChainRepository extends ConfigurableService implements KeyChainRepositoryInterface
+{
+    public const SERVICE_ID = 'taoLti/PlatformKeyChainRepository';
+    public const OPTION_DEFAULT_KEY_ID = 'defaultKeyId';
```
Comment (boajer) [inline models/classes/Security/DataAccess/Repository/PlatformKeyChainRepository.php:38]: Do you really need those constants to be public?

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: test/unit/models/classes/Security/DataAccess/Repository/PlatformJwksRepositoryTest.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            ],
+            json_decode(
+                json_encode(
+                    $this->subject->find()->jsonSerialize()
+                ),
+                true
+            )
+        );
+    }
+}
```
Comment (mike-ionut-mihai-sandu-tao) [inline test/unit/models/classes/Security/DataAccess/Repository/PlatformJwksRepositoryTest.php:104]: \n

Comment (boajer) [inline test/unit/models/classes/Security/DataAccess/Repository/PlatformJwksRepositoryTest.php]: could you run PSR-12 formatter against your code

**Pull request merged**